script/check_format

```diff
@@ -6,5 +6,4 @@ cd "$(dirname "$0")/.."
 black \
   --check \
   --fast \
-  --quiet \
```

Comment:
Please don't change this script.

Reply:
I'm sorry. Modified just for local run, because CI complained, but commited accidentally.